files/.github/workflows/stale.yml

```
@@ -19,7 +19,7 @@ jobs:
         stale-issue-message: 'Oops, seems like we needed more information for this issue, please comment with more details or this issue will be closed in 7 days.'
         close-issue-message: 'This issue was closed because it is missing author input.'
         stale-issue-label: 'kind/stale'
-        any-of-labels: 'hint/needs-author-input,need/author-input'
+        any-of-labels: 'need/author-input'
```

@guseggert @BigLep I propose we exempt `need/author-input` issues from being autoclosed if they are marked with any of the below labels. 

Essentially, if  we mark something as `need/analysis` it won't get autoclosed.

```suggestion
        any-of-labels: 'need/author-input'
        exempt-issue-labels: 'need/maintainer-input,need/maintainers-input,need/analysis'
```

---

It makes sense to me. Let's go with this and we can adjust if we find it to lenient :)

---

Expanded the list with `need/triage` and `need/community-input` in https://github.com/ipfs/github-mgmt/pull/4/commits/019d6790b8e77df3c86a11868419ac51e31578cc